### PR TITLE
Remove search icon in message view

### PR DESCRIPTION
**Pull request by konstantinullrich**
### First time contributor checklist:

* [x] I have read the [README](https://github.com/loki-project/loki-messenger/blob/master/README.md) and [Contributor Guidelines](https://github.com/loki-project/loki-messenger/blob/master/CONTRIBUTING.md)

### Contributor checklist:

* [x] My commits are in nice logical chunks with [good commit messages](http://chris.beams.io/posts/git-commit/)
* [x] My changes are [rebased](https://medium.freecodecamp.org/git-rebase-and-the-golden-rule-explained-70715eccc372) on the latest [`clearnet`](https://github.com/loki-project/loki-messenger/tree/development) branch
* [x] A `yarn ready` run passes successfully ([more about tests here](https://github.com/loki-project/loki-messenger/blob/master/CONTRIBUTING.md#tests))
* [x] My changes are ready to be shipped to users

### Description

Removed the redundant Search Icon in the Message View.
One Test is failing unrelated to the change and I will open an issue for that.

### CODE REVIEW

File: ts/components/conversation/ConversationHeader.tsx [hunk 1]
```diff
@@ -392,12 +379,7 @@ export class ConversationHeader extends React.Component<Props> {
           </div>
           {this.renderExpirationLength()}
 
-          {!this.props.isRss && (
-            <>
-              {this.renderSearch()}
-              {this.renderAvatar()}
-            </>
-          )}
+          {!this.props.isRss && <>{this.renderAvatar()}</>}
```
Comment (Mikunj) [inline ts/components/conversation/ConversationHeader.tsx]:
```suggestion
          {!this.props.isRss && this.renderAvatar()}
```
`<>` were redundant. This should do the same thing. Could you check if this works and also run `yarn lint-full` once done?

Comment (konstantinullrich) [reply ts/components/conversation/ConversationHeader.tsx]: 1 Second

**Comment by konstantinullrich** [comment]
Hey, @Mikunj I implemented your suggestion :)

**Comment by Mikunj** [comment]
Cool, thanks for the PR. I'm going to do a final check through once i am in the office tomorrow before approving and merging :)

**Comment by Haafingar** [comment]
Hey @konstantinullrich please email me at simon@loki.network for full bounty payment. Nice work!

**Comment by konstantinullrich** [comment]
@Haafingar I sent you an Email. :) My Email address is konstantinullrich12@gmail.com

**Pull request merged**